src/components/Tabs/Tabs.stories.tsx

```diff
@@ -108,28 +126,30 @@ export default {
     tags: ['autodocs'],
     argTypes: {
         paddingX: {
-            options: Object.values(TabsPaddingX),
+            options: TABS_PADDING_X_TYPES,
             control: { type: 'select' },
-            defaultValue: TabsPaddingX.Small,
+            defaultValue: '8',
         },
         size: {
-            options: Object.values(TabSize),
+            options: TABS_SIZE_TYPES,
             control: { type: 'select' },
-            defaultValue: TabSize.Small,
+            defaultValue: 'Small',
         },
     },
 } as Meta<TabsProps>;
 
 const TabTemplate: StoryFn<TabsProps> = (args) => {
     const [activeItemId, setActiveItemId] = useState(data[0].id);
     return (
-        <Tabs {...args} activeItemId={activeItemId} onChange={(value) => setActiveItemId(value)}>
-            {data.map((item) => (
-                <TabItem id={item.id} key={item.id} label={item.label} disabled={item.disabled ?? false}>
-                    <div className="tw-p-3">{item.children}</div>
-                </TabItem>
-            ))}
-        </Tabs>
+        <div className="tw-max-h-[200px] tw-flex tw-flex-col">
```

Comment:
I wouldn't enforce the max-height on every story, it's misleading. I would create a story `TabsWithMaxHeight` instead.